app/assets/stylesheets/fae/modules/_modal.scss

```diff
@@ -56,16 +55,20 @@
   img {
     max-width: 100%;
     max-height: 100%;
+    margin: auto;
+    display: block;
   }
 }
 
 .simplemodal-close {
+  display: block;
+  height: 30px;
```

Comment:
Locking in the height and appending `55` to the dimensions in JS may not be the best solution here. Does using `box-sizing: content-box;` work? I'd prefer to use CSS for anything that doesn't dynamically calculate the size of an object.